packages/pwa/app/components/product-scroller/index.jsx

```diff
@@ -98,6 +98,15 @@ const ProductScroller = forwardRef(
                                             {...(typeof productTileProps === 'function'
                                                 ? {...productTileProps(product)}
                                                 : {...productTileProps})}
+                                            imageProps={{
+                                                sizes: {
+                                                    base: '70vw',
+                                                    md: '40vw',
+                                                    lg: '30vw',
+                                                    '2xl': '489px'
+                                                },
+                                                widths: [275, 550, 825, 489, 978]
```

Comment:
I am concerning of passing these numbers as `widths` directly as prop here. They look pretty "magical". 
Can we put them into a style prop and reuse them here? or Is there a way to make them more sense to pass as widhtProps (perhaps making they array into a constant with comment explaining the values)?

Reply:
@alexvuong yeah, I can see how `widths` can be magical 😅 Let me write down what Ben suggested to me earlier during a huddle. It would try to make the interface simpler, so the widths would be automatically calculated for you.